### ISSUE TITLE
Check that we dont have any "Snabbprotokoll"

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#153]
After a discussion with @rbbby we were not 100% sure that there is no snabbprotokoll in the data. Its not a big issue, but we should probably double-check this long term and before a version 1.0.

**Comment by salgo60** [comment]
Example of a "Snabbprotokoll"
* [H90992](https://data.riksdagen.se/dokument/H90992) [json](https://data.riksdagen.se/dokument/H90992.json) [xml](http://data.riksdagen.se/dokumentstatus/H90992) [pdf](http://data.riksdagen.se/fil/5507DADB-6F28-4345-8371-3F0E1D489ECA)
maybe they need another document type...

**Comment by BobBorges** [comment]
What are the main differentiating characteristics between a snabbprotokoll and a (trög?)protokoll?

**Comment by salgo60** [comment]
`Snabbprotokoll`
* [www.synonymer.se](https://www.synonymer.se/sv-syn/snabbprotokoll)

**Comment by fredrik1984** [comment]
@BobBorges in 1952, snabbprotokoll (preliminary record) is introduced. This was a way to publish a not-yet-final-debate-record so that the public/press could be informed about what was said in the riksdag quicker, otherwise it could take several weeks until the final debate record was printed. The final debate record is a bit more thorough proofread etc. In the Swerik project we are working with the final debate record. Hence, in theory, there should be no preliminary record in our material. We are working with Kammarens protokoll: https://www.riksdagen.se/sv/sok/?avd=dokument&doktyp=prot

**Comment by BobBorges** [comment]
That's all clear -- thanks -- I'm thinking about how I can unittest that our protocols are kmmarens and not snabb.

**Comment by fredrik1984** [comment]
Yes, that would be a good idea. @Lottabrorsson do you have a few examples of snabbprotokoll since 1952? Like one from each decade?

**Comment by MansMeg** [comment]
This issue is not part of any milestone so it has (at least now) a lower priority, I would say.

**Comment by fredrik1984** [comment]
True. @BobBorges you can create an issue and we can deal with this later. I actually don't think it is a big problem.

**Comment by MansMeg** [comment]
This is the issue... =)

**Comment by fredrik1984** [comment]
haha, so meta!

**Comment by BobBorges** [comment]
Trying to _remove_ issues here :D

**Comment by BobBorges** [comment]
These appear to be snabb:
```
200405/prot-200405--121.xml
200405/prot-200405--122.xml
200405/prot-200405--144.xml
200708/prot-200708--066.xml
200708/prot-200708--067.xml
200708/prot-200708--068.xml
201011/prot-201011--123.xml
201011/prot-201011--127.xml
201011/prot-201011--128.xml
201213/prot-201213--053.xml
201213/prot-201213--059.xml
201213/prot-201213--075.xml
201213/prot-201213--108.xml
201213/prot-201213--135.xml
201213/prot-201213--136.xml
201314/prot-201314--055.xml
201314/prot-201314--106.xml
201314/prot-201314--108.xml
```

**Comment by Lottabrorsson** [comment]
@BobBorges The PDF:s are correct, protocols. Do you need them as html/xml?

**Comment by MansMeg** [comment]
I guess we should:
- [ ] Fix these records
- [ ] Add a unit test to identify new, obvious, snabbprotokoll

**Comment by BobBorges** [comment]
I can implement this or similar in a unittest (commenting so I don't forget):
![image](https://github.com/welfare-state-analytics/riksdagen-corpus/assets/8887511/ed2e3341-70a5-48d8-8318-ff9f9817c86a)

**Comment by MansMeg** [comment]
Yes. That seem like a good strategy. We just want to capture this if it happens in the future.
